### ISSUE TITLE
a11y - Violation in Utils

### ISSUE THREAD

**Issue by francinelucca** [carbon-design-system/gatsby-theme-carbon#1304]
The Gatsby theme utils wrapping div currently has `aria-label` but no role on it and it is triggering the following violation:
`The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"`

## Steps to reproduce the issue

1. Go to a website that uses Gatsby Utils (i.e.: https://carbondesignsystem.com/)
2. Scroll down the page enough so that you can see the backtotop and feedback buttons
3. Run Accessibility Checker on the page
4.Notice a11y violation

## Additional information
<img width="203" alt="image" src="https://github.com/carbon-design-system/gatsby-theme-carbon/assets/40550942/68ca6776-2b19-456f-8379-d1244b4d361a">
<img width="582" alt="image" src="https://github.com/carbon-design-system/gatsby-theme-carbon/assets/40550942/58a191cb-0324-414c-bd8c-e6479e96cb56">
